core/src/main/java/com/dtolabs/rundeck/core/config/RundeckConfigBase.java

```diff
@@ -542,7 +548,15 @@ public Enabled(final Boolean enabled) {
         @Data
         public static class PaginateJobs {
             Boolean enabled;
-            String maxPerPage;
+            Max max;
+            @Data
+            public static class Max {
+                Per per;
+                @Data
+                public static class Per {
+                    String page;
```

Comment:
should be an `Integer`?

Reply:
Yep, It's obvious it was a copy paste haha